docs/src/main/asciidoc/security-basic-authentication.adoc

```diff
@@ -69,6 +69,33 @@ For more information about how you can secure your Quarkus applications by using
 {project-name} also includes built-in security to allow for role-based access control (RBAC) based on the common security annotations `@RolesAllowed`, `@DenyAll`, `@PermitAll` on REST endpoints and CDI beans.
 For more information, see the Quarkus xref:security-authorize-web-endpoints-reference.adoc[Authorization of web endpoints] guide.
 
+[[basic-auth-programmatic-set-up]]
+== Set up Basic authentication programmatically
+
+The `io.quarkus.vertx.http.security.event.Basic` CDI event allows to customize the configuration provided in the 'application.properties' during the runtime like in the example below:
```

Comment:
```suggestion
The `io.quarkus.vertx.http.security.event.Basic` CDI event allows to configure the Basic authentication mechanism programmatically like in the example below:
```

Same suggestion as with the Form, even if it also works in the customization case as you suggest